Terminal.Gui/ConsoleDrivers/CursesDriver/binding.cs

```diff
@@ -93,7 +92,7 @@ public static int Lines
     //
     public static int addch (int ch)
     {
-        if ((ch < 127) || (ch > 0xffff))
+        if (ch < 127 || ch > 0xffff)
```

Comment:
I think there's a ReSharper rule for these but I'm not certain.

In any case, this sort of thing can be `if (ch is < 127 or > 0xffff)`, which the compiler is capable of optimizing better.